pinot-servicemanager/Dockerfile

```diff
@@ -18,29 +16,29 @@ RUN mkdir classes && cd classes && \
   (for JAR in ../libs/*; do jar -xf $JAR; done) && \
   cd .. && rm -rf libs
 
+ADD install /tmp/install
```

Comment:
curious about ADD vs COPY 🏄

Reply:
On top of that I think it is not too explicit whereas `install` is a file or a directory (because WORKDIR is also called `/install`). Maybe change the name here as `installer` if it is a file?

Reply:
Second thought is that it would be easier if this was calle `install.sh`.

Reply:
I would say it is less distracting to rename the file, yes. As you've thought about this file for a week, heh, it shows renaming the file is a good idea as it doesn't deserve the attention!

Reply:
https://github.com/hypertrace/schema-registry/pull/12 and I'll do this for all images that have a install script, so that the pattern is uniform!

Reply:
https://github.com/hypertrace/kafka/pull/16

Reply:
last one! https://github.com/hypertrace/pinot/pull/31